xiblint/rules/unavailable_classes.py

```diff
@@ -0,0 +1,32 @@
+from xiblint.rules import Rule
+
+
+class UnavailableClasses(Rule):
+    """
+    Ensures a given custom class isn't used system types are subclassed by a set of classes.
+
+    You must specify a module as part of the class name.
+
+    Example configuration:
+    {
+      "classes": {
+          "SomeModule.LegacyButton": "SomeModule.NewButton"
+      }
+    }
+    """
+    def check(self, context):  # type: (Rule, xiblint.xibcontext.XibContext) -> None
+        unavailable_classes = self.config.get('classes', {})
+        if not unavailable_classes:
+            return
```

Comment:
maybe error in this case? no reason to have this rule on if you aren't configuring this